packages/pn-landing-webapp/api/data/perfezionamento.tsx

```diff
@@ -1,188 +1,208 @@
-        subtitle: (
-          <>
-            Dipende dal modo in cui la si riceve: seleziona i canali per sapere
-            quando avviene il perfezionamento.
-          </>
-        ),
-      },
+  {
+    name: "heading title notification viewed 1",
+    data: {
+      title: "Quando si perfeziona una notifica?",
+      subtitle: (
+        <>
+          Dipende dal modo in cui la si riceve: seleziona i canali per sapere
+          quando avviene il perfezionamento.
+        </>
+      ),
     },
-  ];  
-
+  },
+];
 
 const tabs: Array<ITabsData> = [
-    {
-        name: "tabs notification viewed 1",
-        data: {
-            tabs: ["PEC", "Raccomandata", "App IO", "SEND", "Email o SMS"],
-        },
+  {
+    name: "tabs notification viewed 1",
+    data: {
+      tabs: ["PEC", "Raccomandata", "App IO", "SEND", "Email o SMS"],
     },
+  },
 ];
 
 const infoblocks: Array<IInfoblockData> = [
-    {
-        name: "infoblock notification viewed 1",
-        data: {
-            title: "",
-            inverse: false,
-            image: `${IMAGES_PATH}/pf-notification-viewed-1.png`,
-            imageShadow: false,
-            content: (
-                <>
-                    <Typography variant="overline">LA NOTIFICA SI PERFEZIONA</Typography>
-                    <Typography variant="h4" color="primary">
-                        7 giorni
-                    </Typography>
-                    <Box sx={{ pr: { lg: "20%", xs: 0 } }}>
-                        <Typography sx={{ mb: 3 }}>
-                            dopo la data in cui ricevi la PEC. Se la ricevi dalle 21:00 in
-                            poi, conta a partire dal giorno dopo.
-                        </Typography>
-                        <Alert color="info" variant="outlined" icon={<IconInfo />}>
-                            <AlertTitle>La PEC è satura, non valida o inattiva?</AlertTitle>
-                            <Typography variant="body2">
-                                In questo caso, la notifica si perfeziona 15 giorni dopo la data
-                                di deposito dell’avviso di mancato recapito. Lo trovi su SEND,
-                                nel dettaglio della notifica.
-                            </Typography>
-                        </Alert>
-                    </Box>
-                </>
-            ),
-        },
+  {
+    name: "infoblock notification viewed 1",
+    data: {
+      title: "",
+      inverse: false,
+      image: `${IMAGES_PATH}/pf-notification-viewed-1.png`,
+      imageShadow: false,
+      content: (
+        <>
+          <Typography variant="overline">LA NOTIFICA SI PERFEZIONA</Typography>
+          <Typography variant="h4" color="primary">
+            dopo 7 o 15 giorni
+          </Typography>
+          <Box sx={{ pr: { lg: "20%", xs: 0 } }}>
+            <List sx={{ listStyleType: "disc", pl: 4 }}>
+              <ListItem sx={{ display: "list-item", pl: 0 }}>
```

Comment:
On Figma the list items are more dense, so put for **each** ListItem
`pl:0, pt:0, pb:0.5`